package.json

```diff
@@ -81,11 +85,19 @@
     "react-tools": "0.13.3",
     "react-addons-css-transition-group": "0.14.8",
```

Comment:
Wouldn't this dependency conflict with the current react v15.x?

Reply:
Doesn't conflict and it's a dependency I hope we can remove soon.